crates/table/Cargo.toml

```diff
@@ -29,8 +29,8 @@ proptest = ["dep:proptest", "dep:proptest-derive"]
 blake3_pure = ["blake3/pure"]
 
 [dependencies]
-spacetimedb-primitives = { path = "../primitives" }
-spacetimedb-sats = { path = "../sats" }
```

Comment:
I updated these to use the new workspace thing because cargo was complaining that these didn't specify version numbers